src/status_im2/contexts/communities/menus/community_options/__tests__/community_options_component_spec.cljs

```diff
@@ -0,0 +1,202 @@
+
+(ns status-im2.contexts.communities.menus.community-options.--tests--.community-options-component-spec
+  (:require ["@testing-library/react-native" :as rtl]
+            [re-frame.core :as re-frame]
+            [status-im2.contexts.communities.menus.community-options.view :as options]
+            [reagent.core :as reagent]))
+
+
+(defn setup-sub
+  [opts]
+  (re-frame/reg-sub
+   :communities/community
+   (fn [_] opts)))
+
+(defn render-options-bottom-sheet
+  []
+  (rtl/render (reagent/as-element [options/community-options-bottom-sheet {:id "test"}])))
+
+(js/global.test "joined options - Non token Gated"
```

Comment:
We can use some helpers from the namespace `test-helpers.component`. In particular, the macros `test-helpers.component/describe` and `test-helpers.component/test` can help with test readability by grouping related tests, instead of doing what you did here to start the test descriptions with a common prefix, like `joined options - `.

Reply:
yep, will update this file to the new helpers!